project/plugins.sbt

```diff
@@ -5,12 +5,11 @@ val scalaJSVersion = Option(System.getenv("SCALAJS_VERSION")).getOrElse("0.6.19"
 addSbtPlugin("com.jsuereth"      % "sbt-pgp"                  % "1.0.1")
 addSbtPlugin("com.github.gseitz" % "sbt-release"              % "1.0.5")
 addSbtPlugin("org.scala-js"      % "sbt-scalajs"              % scalaJSVersion)
-addSbtPlugin("org.scala-native"  % "sbt-crossproject"         % "0.2.2")
-addSbtPlugin("org.scala-native"  % "sbt-scala-native"         % "0.3.3")
+addSbtPlugin("org.scala-native"  % "sbt-scala-native"         % "0.3.3" exclude("org.scala-native", "sbt-crossproject"))
 
 {
-  if (scalaJSVersion.startsWith("0.6."))
-    Seq(addSbtPlugin("org.scala-native" % "sbt-scalajs-crossproject" % "0.2.2"))
-  else
+  if (scalaJSVersion == "1.0.0-M1")
```

Comment:
`1.0.0-M2`?

Reply:
No this is specifically for 1.0.0-M1, because it included the contents of `sbt-scalajs-crossproject`. In 1.0.0-M2, sbt-scalajs does not provide `JSPlatform` anymore, and people must explicitly use `sbt-scalajs-crossproject`.

Moreover, in 0.6.x, although sbt-scalajs provided a `crossProject`, it was not compatible with Scala Native. So in 0.6.x we must also force `sbt-scalajs-crossproject`.

This leads to this funny situation where sbt-scalajs-crossproject must be included for 0.6.x and 1.0.0-M2+, but not for 1.0.0-M1.